codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Headers.java

```diff
@@ -184,6 +186,14 @@ public static boolean isPseudoHeader(ByteString header) {
     Http2Headers clear();
 
     /**
+     * Returns an iterator over all HTTP/2 headers from this instance. The iteration order is as follows
+     *   1. All reserved headers come first (in no particular order).
```

Comment:
I think the term the spec uses for these is [Pseudo-Header](https://tools.ietf.org/html/draft-ietf-httpbis-http2-17#section-8.1.2.3).